PIL/Image.py

```diff
@@ -1519,7 +1519,7 @@ def putpixel(self, xy, value):
             return self.pyaccess.putpixel(xy, value)
         return self.im.putpixel(xy, value)
 
-    def resize(self, size, resample=NEAREST):
+    def resize(self, size, resample=NEAREST, roi=None):
```

Comment:
I like box for a parameter name. This parameter runs the equivalent of `resize(crop(box))`, so matching the `Image.crop` name makes sense.